<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageQt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="default.qrc">
    <normaloff>:/img/src/logo_2.png</normaloff>:/img/src/logo_2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <layout class="QHBoxLayout" name="leftButtonLayout">
          <item>
           <widget class="QPushButton" name="openBtn">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeBtn">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveAsBtn">
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="hstgrmBtn">
            <property name="text">
             <string>Histogram</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="normalBtn">
            <property name="text">
             <string>Right Normal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="GraphicsView" name="leftGraphicsView" native="true"/>
        </item>
        <item>
         <widget class="GraphicsView" name="rightGraphicsView" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionAdjust_brightness"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="actionHistogram"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionChinese"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionAbout"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuGrey_Transfrom">
    <property name="title">
     <string>&amp;Grey Transfrom</string>
    </property>
    <addaction name="actionLinear_level_transformation"/>
    <addaction name="actionPower_transformation"/>
    <addaction name="actionLogarithm_grey_level_transformation"/>
    <addaction name="actionExp_transfrom"/>
    <addaction name="actionTwo_thresholds_transform"/>
    <addaction name="actionStretch_transformation"/>
   </widget>
   <widget class="QMenu" name="menuGeometric">
    <property name="title">
     <string>Geo&amp;metric Transform</string>
    </property>
    <widget class="QMenu" name="menuRotate_2">
     <property name="title">
      <string>&amp;Rotate</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
    </widget>
    <widget class="QMenu" name="menuFlip_2">
     <property name="title">
      <string>&amp;Flip</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
    </widget>
    <addaction name="menuRotate_2"/>
    <addaction name="zoomAction"/>
    <addaction name="menuFlip_2"/>
   </widget>
   <widget class="QMenu" name="menuArtistic_Effect">
    <property name="title">
     <string>Artistic Effect</string>
    </property>
    <widget class="QMenu" name="menuFrame">
     <property name="title">
      <string>&amp;Frame</string>
     </property>
     <addaction name="actionMovie_frame"/>
     <addaction name="actionClassic_frame"/>
     <addaction name="actionFlower_frame"/>
    </widget>
    <widget class="QMenu" name="menuTexture">
     <property name="title">
      <string>&amp;Texture</string>
     </property>
     <addaction name="actionMetal"/>
    </widget>
    <widget class="QMenu" name="menuTemperature">
     <property name="title">
      <string>T&amp;emperature</string>
     </property>
     <property name="icon">
      <iconset resource="default.qrc">
       <normaloff>:/img/src/temperature_72px.png</normaloff>:/img/src/temperature_72px.png</iconset>
     </property>
     <addaction name="actionCool"/>
     <addaction name="actionWarm"/>
    </widget>
    <addaction name="menuFrame"/>
    <addaction name="menuTexture"/>
    <addaction name="menuTemperature"/>
   </widget>
   <widget class="QMenu" name="menuBlur">
    <property name="title">
     <string>Smoothi&amp;ng</string>
    </property>
    <addaction name="actionSimple"/>
    <addaction name="actionGauss"/>
    <addaction name="separator"/>
    <addaction name="actionMeida_Filter"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionAdjust"/>
    <addaction name="actionNormal"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuArtistic_Effect"/>
   <addaction name="menuBlur"/>
   <addaction name="menuGeometric"/>
   <addaction name="menuGrey_Transfrom"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTest"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/document-open.png</normaloff>:/img/src/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/save.svg</normaloff>:/img/src/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Sa&amp;ve As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/avatar-default.png</normaloff>:/img/src/avatar-default.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionAdjust">
   <property name="text">
    <string>Adjust</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/restore.png</normaloff>:/img/src/restore.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;estore</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/left.png</normaloff>:/img/src/left.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/right.png</normaloff>:/img/src/right.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Right</string>
   </property>
  </action>
  <action name="zoomAction">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/zoom.png</normaloff>:/img/src/zoom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>G&amp;rayscale</string>
   </property>
  </action>
  <action name="actionWarm">
   <property name="text">
    <string>&amp;Warm</string>
   </property>
  </action>
  <action name="actionCool">
   <property name="text">
    <string>&amp;Cool</string>
   </property>
  </action>
  <action name="actionMovie_frame">
   <property name="text">
    <string>&amp;Movie frame</string>
   </property>
  </action>
  <action name="actionClassic_frame">
   <property name="text">
    <string>&amp;Classic frame</string>
   </property>
  </action>
  <action name="actionFlower_frame">
   <property name="text">
    <string>&amp;Flower frame</string>
   </property>
  </action>
  <action name="actionMetal">
   <property name="text">
    <string>&amp;Metal</string>
   </property>
  </action>
  <action name="actionAdjust_brightness">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/brightness.svg</normaloff>:/img/src/brightness.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Adjust brightness</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/chinese.png</normaloff>:/img/src/chinese.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/english.png</normaloff>:/img/src/english.png</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/horizontal.png</normaloff>:/img/src/horizontal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/vertical.png</normaloff>:/img/src/vertical.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Vertical</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/histogram.png</normaloff>:/img/src/histogram.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Histogram</string>
   </property>
  </action>
  <action name="actionLinear_level_transformation">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/line_graphic_72p.png</normaloff>:/img/src/line_graphic_72p.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Linear transform</string>
   </property>
  </action>
  <action name="actionLogarithm_grey_level_transformation">
   <property name="text">
    <string>L&amp;ogarithm transform</string>
   </property>
  </action>
  <action name="actionPower_transformation">
   <property name="text">
    <string>&amp;Power transform</string>
   </property>
  </action>
  <action name="actionExp_transfrom">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/Line_Chart_72px.png</normaloff>:/img/src/Line_Chart_72px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exp transfrom</string>
   </property>
  </action>
  <action name="actionTwo_thresholds_transform">
   <property name="text">
    <string>&amp;Two thresholds transform</string>
   </property>
  </action>
  <action name="actionStretch_transformation">
   <property name="text">
    <string>&amp;Stretch transformation</string>
   </property>
  </action>
  <action name="actionSimple">
   <property name="text">
    <string>&amp;Simple</string>
   </property>
  </action>
  <action name="actionGauss">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/blur_on_72px.png</normaloff>:/img/src/blur_on_72px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Gauss</string>
   </property>
  </action>
  <action name="actionMeida_Filter">
   <property name="text">
    <string>Meida Filter</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionT">
   <property name="text">
    <string>t</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QWidget</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="default.qrc"/>
 </resources>
 <connections/>
</ui>
